AdobeStockAsset/Test/_files/api_assets.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
I don't think we need this fixture as there is more advanced `AdobeStockAsset/Test/_files/asset.php` introduced